docs/guides/Integration-Guides/headless-commerce/headless-authentication.md

```diff
@@ -0,0 +1,87 @@
+
+>❗ This feature is currently being tested by a limited number of VTEX accounts. Please do not share this documentation with people outside of your operation.
+
+## Implementation
+
+This is the flow of shopper authentication for headless VTEX stores:
+
+![headless authentication flow](./headless-authentication_1.png)
+
+1. The shopper authenticates themselves in your frontend via an [OAuth integration](#oauth-log-in).
+2. Your frontend [exchanges the OAuth identity provider access token for a VTEX user token](#exchanging-oauth-identity-provider-access-token-for-vtex-user-token).
+3. Now you can make requests from your frontend to VTEX APIs using the obtained [user token](https://developers.vtex.com/docs/guides/getting-started-authentication#user-token).
+
+Below you can learn more details on each of these steps.
+
+>❗ If you are building a headless store, it means you are using a frontend application different from VTEX's native frontend. Because of this, it is important to follow the instructions provided below and be aware of your [application's responsibilities](#authentication-responsibilities-of-headless-applications).
+
+### OAuth log in
+
+Shopper authentication in your headless store must happen through an external identity provider. To do this, you must [set up an OAuth login integration](https://developers.vtex.com/docs/guides/login-integration-guide-webstore-oauth2).
+
+After you have configured your [OAuth](https://developers.vtex.com/docs/guides/login-integration-guide-webstore-oauth2) connection in the VTEX Admin panel, you can implement the OAuth login on your frontend, according to your store's needs and [responsibilities](#authentication-responsibilities-of-headless-applications).
+
+>⚠️ Make sure that the [OAuth access token](https://developers.vtex.com/docs/guides/login-integration-guide-webstore-oauth2#relevant-requests) has scope `email` (or another value associated with scope `email`).
+
+### Exchanging OAuth identity provider access token for VTEX user token
+
+If you have set up your OAuth integration and implemented login on your frontend, users may be able to authenticate themselves, but this is still not enough for your frontend to communicate with VTEX REST APIs.
+
+To be able to do this, your frontend must exchange the [access token](https://developers.vtex.com/docs/guides/login-integration-guide-webstore-oauth2#relevant-requests) from the OAuth login for a VTEX [user token](https://developers.vtex.com/docs/guides/getting-started-authentication#user-token) via the [Exchange OAuth access token for VTEX credential API request](https://developers.vtex.com/docs/api-reference/vtex-id-api#post-/vtexid/audience/webstore/provider/oauth/exchange). See an example below:
```

Comment:
```suggestion
To enable this communication, your frontend application must exchange the [access token](https://developers.vtex.com/docs/guides/login-integration-guide-webstore-oauth2#relevant-requests) from the OAuth login for a VTEX [user token](https://developers.vtex.com/docs/guides/getting-started-authentication#user-token). This can be achieved through the [POST - Exchange OAuth access token for VTEX credential](https://developers.vtex.com/docs/api-reference/vtex-id-api#post-/vtexid/audience/webstore/provider/oauth/exchange) API.

See an example below of how to use the **POST - Exchange OAuth access token for VTEX credential** API to perform this operation:
```

Reply:
Sugestão para deixar mais explícito que o link é de um endpoint e para deixar mais explícito o que é "this" em "to be able to do this"